apps/web/src/modules/create-proposal/hooks/useAvailableUpgrade.ts

```diff
@@ -210,11 +205,10 @@ export const useAvailableUpgrade = (
   ): Proposal | undefined => {
     const activeProposals = proposals?.filter(
       (proposal) =>
-        proposal.status === NounsProposalStatus.Active ||
-        proposal.status === NounsProposalStatus.Pending ||
-        proposal.status === NounsProposalStatus.Queued ||
-        proposal.status === NounsProposalStatus.Succeeded ||
-        proposal.status === NounsProposalStatus.Executable
```

Comment:
why did we remove the executable state here?

Reply:
since the subgraph cannot reliably manage proposal status since it changes at specific timestamps I've refactored the proposal code to pull that from the contract. since contract does not include executable as a proposal state it has been removed from status checks (the proposal is still in queued status so the checks still function properly) and the proper code to check the executability of the proposal has been added where necessary @nadiakiamilev .